### PR TITLE
Enabling/Disabling of components

### LINKED ISSUE #1095: disable/hide component
Add a new status or a property to a component so it can be disabled and/or made invisible. 
Helps in the preparation stage, when you first prepare placeholders for all the required checks and then start connecting real checks and enabling components.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #1095

Ping @cachethq/owners in case.

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Fin. Let me know.

File: app/Composers/StatusPageComposer.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function compose(View $view)
                 ];
             }
         } else {
-            if (Component::whereIn('status', [2, 3])->count() > 0) {
+            if (Component::where('enabled', true)->whereIn('status', [2, 3])->count() > 0) {
```
Comment (GrahamCampbell) [inline app/Composers/StatusPageComposer.php]: better to have an enabled scope on the component model surely?

Comment (GrahamCampbell) [inline app/Composers/StatusPageComposer.php]: Like these:

image

Comment (jbrooksuk) [inline app/Composers/StatusPageComposer.php]: Yeah, I'll do this now.

Comment (jbrooksuk) [inline app/Composers/StatusPageComposer.php]: Scoped.

**Comment by jbrooksuk** [comment]
Squashed into one commit.

**Pull request merged**